### PR TITLE
Added Instagram & Github Links

### PR DESCRIPTION
**Pull request by MarcusSanAntonio**
Removed old link, added instagram and github

### CODE REVIEW

File: src/_components/Footer.jsx [hunk 1]
```diff
@@ -9,10 +9,9 @@ export default function Footer() {
     const apply_url = "https:///sitblueprint.com/apply";
     const process_url = "https:///sitblueprint.com/process";
     const faq_url = "https:///sitblueprint.com/faq";
-    const facebook_url = "https://www.facebook.com/stevensblueprint";
-    const twitter_url = "https://twitter.com/stevensblueprint";
-    const youtube_url = "https://www.youtube.com/channel/UCZ3Z4Z6qZ3Z4Z6qZ3Z4Z6qQ";
     const linkedin_url = "https://www.linkedin.com/company/stevens-blueprint/";
+    const github_url = "https://github.com/stevensblueprint";
+    const instagram_url = "https://instagram.com/stevensblueprint?igshid=OGQ5ZDc2ODk2ZA==";
```
Comment (ezrizhu) [inline src/_components/Footer.jsx]: please remove the `?igshid=` part.

**Review by ezrizhu — CHANGES_REQUESTED** [review]
nice work, one minor fix and we're good.

**Review by ezrizhu — APPROVED** [review]
lg2m

**Review by ShawnAviles — APPROVED** [review]
LGTM 👍

**Pull request merged**